app/collections/Products.coffee

```diff
@@ -19,6 +19,9 @@ module.exports = class Products extends CocoCollection
       countrySpecificProduct = @findWhere { name: "#{user?.get('country')}_basic_subscription" }
     return countrySpecificProduct or @findWhere({ name: 'basic_subscription' })
 
+  getBasicYearlySubscriptionForUser: () ->
```

Comment:
If we have taxonomy for `Annual`, does it make sense to update that all the places we can (i.e. everywhere except for hard values like plan ID's)?

Reply:
Great idea. Updating